infrastructure/iac/types.go

```diff
@@ -40,3 +40,17 @@ type iacIssue struct {
 	Documentation  lsp.Uri        `json:"documentation"`
 	IacDescription iacDescription `json:"iacDescription"`
 }
+
+type IssueData struct {
```

Comment:
I suggest you comment the properties with examples on what they contain  🙏

Reply:
Good idea!